I.write-tests/06-largest-number/largest-number.test.js

```diff
@@ -1,5 +1,14 @@
 // example
-// input: [3, 21, 88, 4, 36];
+// input:[3, 21, 88, 4, 36] ;
 // expected: 88;
+// also test that the original array hasn't changed
+var biggestNumber = require("./largest-number");
+test("Get Largest number", function() {
+  // Given
+  var input = [3, 21, 88, 4, 36];
```

Comment:
This is a good example of a case where the test was not sufficient! You need more test cases to be sure that the function is correctly implemented in all cases. Can you add more tests to find out that it is failing with certain arrays?